src/components/ImageList.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import {connect} from 'react-redux'
+import PropTypes from "prop-types";
+
+const IMAGE_HEIGHT = 200; //todo: clarify how to define common constants
```

Comment:
IMO the `ImageList` component should be removed (please see [this comment](https://github.com/nehippi/react-learn/pull/3/files#r963561942)), but JFYI duplicating constants like this is a bad practice. You could either export this constant _here_ and import it in the `ImageListContainer` component, or create a _separate_ file (like `constants.js`) and export the constant from there.

Reply:
Resolved.